src/redefined.resolver.ts

```diff
@@ -112,4 +114,8 @@ export class RedefinedResolver {
     static createSidArbNovaResolver(node?: string) {
         return new SidResolverService(node || config.SID_ARB_ONE_NODE, SidChainId.ARB, "arbitrum-nova", SidResolverData.ARB_NOVA);
     }
+
+    static createBonfidaResolver() {
```

Comment:
@Rednoll 
Perhaps we need to allow the user to specify a cluster

Reply:
done